main.go

```diff
@@ -136,7 +136,7 @@ func main() {
 		proxy.Transport = http.DefaultTransport
 	}
 
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+	http.HandleFunc("/receive", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why `/receive`?

Reply:
Good question. We didn't want to use the root URL, because that's too generic. Also in the future there might be additional routes for metrics, liveliness, etc.

Reply:
But maybe it's better to call it `/write`, indicating that this is writing stuff to Observatorium (in contrast to reading only which uses a different SA).

Reply:
> We didn't want to use the root URL, because that's too generic. 

That's fine as it is a proxy - so having an endpoint also means that this has to be included when someone is configuring his service to target this proxy.  

>  Also in the future there might be additional routes for metrics, liveliness, etc.

I think these could go onto a different port.